docs/qa/application-inventory.md

```
@@ -0,0 +1,800 @@
+| 3 | **Create resume → edit → compile → PDF** | `/workspace/new` or `/workspace` → `POST /resumes/` → `/workspace/[id]/edit` (LaTeXEditor) → `POST /compile` / `/jobs/submit` → `latex_worker` → `/download/{job_id}` PDF |
+| 4 | **AI optimize → review → ATS** | `/workspace/[id]/optimize` → `GuidedIntakePanel` → `POST /optimize` (`require_feature(llm_optimize)`) → `llm_worker` → `ChangeReviewModal` (`/optimize/segment-changes` + `/apply-changes`) → `POST /ats/score` |
+| 5 | **Deep ATS + semantic match** | ATS panels → `POST /ats/deep-analyze` (`ats_worker`) → `POST /ats/semantic-match` (embeddings) → score history |
+| 6 | **Quick tailor to JD** | `QuickTailorModal` → `POST /resumes/{id}/quick-tailor` (fork + optimize job, quota) |
+| 7 | **Batch tailor** | `/workspace/[id]/batch-tailor` → `POST /jobs/batch` (`require_feature(batch_tailor)`) → poll `/jobs/batch/{id}` |
+| 8 | **Templates → resume** | `/templates` (public gallery) → `TemplatePreviewModal` → login → `POST /templates/{id}/use` (`require_feature(templates)`) |
+| 9 | **Guided builder** | `/workspace/builder/new` → `POST /resumes/builder/seed-upload` → `PATCH /resumes/{id}/builder` → `POST /resumes/builder` → compile |
+| 10 | **Import external projects** | `ImportProjectsModal` → GitHub (`/github/import-projects` → `github_import_worker`) / URL (`/sources/import-url`) / LinkedIn (`/sources/import-linkedin`) |
+| 11 | **Import from builder platform** | `ImportFromBuilderWizard` (4 steps) → `/formats/parse` → `/formats/upload` → `converter_worker` → LaTeX |
+| 12 | **Cover letter** | `/workspace/[id]/cover-letter` → `POST /cover-letters/generate` (`cover_letter_worker`) → `/workspace/cover-letters` list |
+| 13 | **Job tracker CRUD** | `/tracker` (kanban) → `AddApplicationModal` → `POST /tracker/applications` (`require_feature(application_tracker)`) → status transitions → `/tracker/stats` |
+| 14 | **One-click apply** | `ApplyModal` → `/apply/detect` → `/apply/{greenhouse|lever}/preview` → `POST /apply/{platform}` (`require_feature(one_click_apply)`) → `/apply/submissions` |
```

markdownlint-cli2 (0.23.2)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)